frontend/src/components/FilterDrawer.vue

```diff
@@ -212,6 +219,47 @@ const checkSearchTerm = (searchTerm, fieldsToSearch, feature) => {
   })
 }
 
+const logIdentifiers = async () => {
+  const selectedIdentifiers = {
+    selectedProcesses: '',
+    selectedSpatialZones: '',
+    selectedTemporalZones: ''
+  }
+
+  const collectIdentifiersAsString = async (ids, map, category) => {
+    const identifiers = []
+    for (let id of ids) {
+      await nextTick()
+      const selectedItem = map.get(id)
+      if (selectedItem) {
+        identifiers.push(selectedItem)
+      } else {
+        console.log(`${category} not found for ID:`, id)
+      }
+    }
+    selectedIdentifiers[category] = identifiers.join(', ') // Join as a string
```

Comment:
seems safe as long as none of our identifiers are ever going to contain a `,`
might be safest to use `|` as a delimiter...
```suggestion
    selectedIdentifiers[category] = identifiers.join('|') // Join as a string
```
might be best to not include a space to allow for easier parsing later